src/helpers/themes/globalStyles.css

```diff
@@ -221,6 +230,27 @@ dialog {
   }
 }
 
+[popover]:not(dialog) {
+  position: relative;
+
+  /* Make the popover use the animation where it grows/shrinks when shown/hidden */
+  animation:
+    scale-in 0.1s forwards,
+    fade-in 0.2s forwards;
```

Comment:
not a fan of random magic animations: should  be variables so they're consistent when we start using them more?

not required for this pr